plugins/exercism/exercism.go

```diff
@@ -0,0 +1,25 @@
+package exercism
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/needsauth"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+)
+
+func ExercismCLI() schema.Executable {
+	return schema.Executable{
+		Name:    "Exercism CLI",
+		Runs:    []string{"exercism"},
+		DocsURL: sdk.URL("https://exercism.org/docs/using/solving-exercises/working-locally"),
+		NeedsAuth: needsauth.IfAll(
```

Comment:
I think there may be some other situations as well when we don't want to prompt, for example for a command such as `exercism configure --token=<your-api-token>`. 

Is `--token` a flag available for all the commands? In that case, we may want to exclude this flag from prompting altogether.